cfme/infrastructure/virtual_machines.py

```diff
@@ -62,100 +65,30 @@
 
 retire_remove_button = "//span[@id='remove_button']/a/img"
 
-vm_templates_tree = partial(accordion.tree, "VMs & Templates")
-vms_tree = partial(accordion.tree, "VMs")
-templates_tree = partial(accordion.tree, "Templates")
-
-drift_table = CheckboxTable({
-    version.LOWEST: "//table[@class='style3']",
-    "5.4": "//th[normalize-space(.)='Timestamp']/ancestor::table[1]"
-})
-
-
-@extend_nav
-class infrastructure_virtual_machines:
-    def infra_vm_and_templates(_):
-        vm_templates_tree("All VMs & Templates")
-
-    def vm_templates_provider_branch(ctx):
-        vm_templates_tree("All VMs & Templates", ctx["provider_name"])
-
-    def datacenter_branch(ctx):
-        vm_templates_tree("All VMs & Templates", ctx["provider_name"], ctx["datacenter_name"])
-
-    def infra_vm_obj(ctx):
-        vm_templates_tree(
-            "All VMs & Templates", ctx["provider_name"], ctx["datacenter_name"], ctx["vm_name"])
-
-    def vm_templates_archived_branch(ctx):
-        vm_templates_tree("All VMs & Templates", "<Archived>")
-
-    def infra_archive_obj(ctx):
-        vm_templates_tree("All VMs & Templates", "<Archived>", ctx["archive_name"])
-
-    def vm_templates_orphaned_branch(ctx):
-        vm_templates_tree("All VMs & Templates", "<Orphaned>")
+match_page = partial(match_location, controller='vm_infra', title='Virtual Machines')
 
-    def infra_orphan_obj(ctx):
-        vm_templates_tree("All VMs & Templates", "<Orphaned>", ctx["orphan_name"])
 
-    class infra_vms:
-        def navigate(_):
-            vms_tree("All VMs")
+def nav_to_vms():
+    from cfme.web_ui.menu import nav
+    nav._nav_to_fn('Compute', 'Infrastructure', '/vm_infra/explorer')(None)
```

Comment:
Last time when I used text for host menu item, psav corrected me and put href instead of text.
Probably this makes sense